inhibit/inhibit.go

```diff
@@ -145,7 +143,7 @@ func (ih *Inhibitor) Mutes(lset model.LabelSet) bool {
 	for _, r := range ih.rules {
 		// Only inhibit if target matchers match but source matchers don't.
 		if inhibitedByFP, eq := r.hasEqual(lset); !r.SourceMatchers.Match(lset) && r.TargetMatchers.Match(lset) && eq {
-			ih.marker.SetInhibited(fp, fmt.Sprintf("%d", inhibitedByFP))
+			ih.marker.SetInhibited(fp, fmt.Sprintf("%s", inhibitedByFP.String()))
```

Comment:
this can just be `ih.marker.SetInhibited(fp, inhibitedByFP.String())`

Reply:
It also appears that the original code works as well?

Reply:
Done. Yes the original code works but the API returns different fingerprint formats between silenced and inhibited alerts:

```
      ...snip...
      "status": {
        "state": "suppressed",
        "silencedBy": [
          "95c9789c-f7f6-4d57-99d3-66c9eb25a4b8"
        ],
        "inhibitedBy": []
      },
      ...snip..
```

```
      ...snip...
      "status": {
        "state": "suppressed",
        "silencedBy": [],
        "inhibitedBy": [
          "12124264373735012735"
        ]
      },
      ...snip...
```